tests/domain/test_message_service.py

```
@@ -0,0 +1,472 @@
+import pytest
+from unittest.mock import patch, MagicMock, mock_open
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)